pandas/core/window/ewm.py

```diff
@@ -71,30 +72,48 @@ class EWM(_Rolling):
 
     Notes
     -----
-    Exactly one of center of mass, span, half-life, and alpha must be provided.
-    Allowed values and relationship between the parameters are specified in the
-    parameter descriptions above; see the link at the end of this section for
-    a detailed explanation.
-
-    When adjust is True (default), weighted averages are calculated using
-    weights (1-alpha)**(n-1), (1-alpha)**(n-2), ..., 1-alpha, 1.
-
-    When adjust is False, weighted averages are calculated recursively as:
-       weighted_average[0] = arg[0];
-       weighted_average[i] = (1-alpha)*weighted_average[i-1] + alpha*arg[i].
-
-    When ignore_na is False (default), weights are based on absolute positions.
-    For example, the weights of x and y used in calculating the final weighted
-    average of [x, None, y] are (1-alpha)**2 and 1 (if adjust is True), and
-    (1-alpha)**2 and alpha (if adjust is False).
-
-    When ignore_na is True (reproducing pre-0.15.0 behavior), weights are based
-    on relative positions. For example, the weights of x and y used in
-    calculating the final weighted average of [x, None, y] are 1-alpha and 1
-    (if adjust is True), and 1-alpha and alpha (if adjust is False).
-
-    More details can be found at
-    https://pandas.pydata.org/pandas-docs/stable/user_guide/computation.html#exponentially-weighted-windows
+    Exactly one paramter: ``com``, ``span``, ``halflife``, or ``alpha``
+    must be provided.
+    Allowed values and the relation between the parameters are specified in the
+    parameter descriptions above (see the link at the end of this section for
+    a detailed explanation).
+
+    Available EW (exponentially weighted) methods: ``mean()``, ``var()``, ``std()``,
+    ``corr()``, ``cov()``.
+
+    When ``adjust=True`` (default), the EW function is calculated using
```

Comment:
Is it possible to have this in the `adjust` parameter description instead?

Reply:
There are 2 cases there which require further details. So I think there is no need to add that to parameter description.

Reply:
Not a big deal, but I don't understand the reasoning. What I have in mind is that if I'm interested in what the `adjust` parameter does, I'll go to its description to understand what it does. And the description there is somehow poor, while here he have all the relevant information.

So, this feels misplaced to me. Not sure how the fact that this happens twice in this page changes this reasoning. Am I missing something?

Reply:
For me, the parameter descriptions are useful when I need to check briefly what parameters may be relevant, what they do, what possible values can they take (via Shift+Tab).
The description needs to be concise, if you want more details about each case, scroll a little below and look at the Notes.

To be able to read the equations there you will have to open the browser. When someone will want to quickly preview the description with Shift+Tab, it will just be cluttered with sphinx signs.

Reply:
I understand how you use the docs, but in the link I gave you explains what's [our convention](https://pandas.pydata.org/pandas-docs/stable/development/contributing_docstring.html) regarding the sections, so I prefer to follow that, and have the parameters sections for the parameters, and the notes for the implementation notes.